dspace-api/pom.xml

```diff
@@ -738,6 +738,17 @@
             <artifactId>jackson-annotations</artifactId>
             <version>2.7.0</version>
         </dependency>
+
+        <dependency>
+            <groupId>org.json</groupId>
+            <artifactId>json</artifactId>
+            <version>20180130</version>
+        </dependency>
+        <dependency>
+            <groupId>io.swagger</groupId>
+            <artifactId>swagger-annotations</artifactId>
+            <version>1.5.12</version>
+        </dependency>
```

Comment:
Are these two new dependencies only added cause of the newly embedded ORCID source code?  Just trying to understand where they are used, especially since we don't use Swagger in DSpace source code.

Reply:
These are not both added just for the newly embedded ORCID source code.
The JSON dependency is used in the new Orcidv2 class to parse the response of the authentication and retrieve the accesstoken that is later used.

The swagger dependency however, IS only used in the "ORCID" code. This contains some of the annotations that are used in these classes.

Reply:
Thanks for clarification, @jonas-atmire .  I was more concerned about the `swagger` dependency. So, it's good to know that's being pulled in just for the ORCID source code.